fastlane/Fastfile

```diff
@@ -68,27 +70,50 @@ lane :release do |options|
   end
 
   sh("git commit -m 'Bump #{version_number}'")
-  sh("git tag #{version_number}")
 
   if(!prompt(text: "Will push changes. All looking good?", boolean: true))
     UI.user_error!("Not pushing changes")
   end
 
-  push_to_git_remote(tags: true)
+  push_to_git_remote(tags: false)
+
+  # Open a new PR
+  create_pull_request(
+    api_token: ENV["GITHUB_TOKEN"],
+    repo: "GetStream/stream-chat-swift",
+    title: "#{version_number} Release",
+    head: "release/#{version_number}",
+    base: "main",
+    body: "#{changes}"
+)
 
+  UI.success("Successfully started release #{version_number}")
+end
+
+desc "Completes an SDK Release"
+lane :publish_release do |options|
+  version = options[:version]
   github_release = set_github_release(
                      repository_name: "GetStream/stream-chat-swift",
                      api_token: ENV["GITHUB_TOKEN"],
-                     name: version_number,
-                     tag_name: version_number,
+                     name: version,
+                     tag_name: version,
                      description: changes
                    )
-
-  # The & operator makes sure truthy values are converted to bool true
+  # The & operator makes sure truthy values are converted to bool
   # and falsy (false and nil) values are converted to bool false
   push_pods(sync: options[:sync] & true)
 
-  UI.success("Successfully released #{version_number}")
+  # Set tag
+  sh("git tag #{version}")
+
+  # Push tag to remote
+  push_to_git_remote(tags: true)
+
+  # Create TestFlight Build
+  testflight_build
```

Comment:
We shouldn't have this, this action runs automatically whenever a release is created, which we do in live 96